src/App.tsx

```diff
@@ -1,18 +1,45 @@
 import React from 'react';
 
 import './App.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { Tabs } from './components/Tabs';
 
 const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+interface State {
+  selectedTab: Tab;
+}
+
+class App extends React.Component<{}, State> {
+  state: State = {
+    selectedTab: tabs[0],
+  };
+
+  changeTab = (tabId: string) => {
+    const currentTab = tabs.find(tab => tab.id === tabId) || tabs[0];
```

Comment:
you can use here findIndex method. Then no need to do 'find' again in a Tabs component, just use index